lib/std/c.zig

```diff
@@ -39,6 +39,33 @@ pub fn versionCheck(comptime glibc_version: std.SemanticVersion) type {
     };
 }
 
+/// Compatibility function, used by Zig's implementation of libc in `lib/c.zig`.
+/// Assumes that `a` and `b` are multi-item pointers with NUL-termination.
```

Comment:
I think this line is unnecessary, it just restates information from the type signature in English.

Reply:
done, sorry for fragmented rebase, I'm a bit confused with my branches